scrapy_azure_exporter/azure_exporter.py

```diff
@@ -90,24 +95,39 @@ def _store_in_thread(self, file):
     def parse_azure_uri(cls, uri):
         "Takes in uri and extracts container name and filename"
         extracted_params = {}
+
         try:
             parsed_url = urlparse(uri)
             # split till 2nd occurrence of slash,
-            # so we can get container name and file name seperated.
+            # so we can get container name and file name separated.
+            # NOTE: This workaround is necessary for compatibility with
+            # Azurite that have the path in the format
+            # `/<account_name>/<container_name>/<filename.ext>`
             splitted = parsed_url.path.split("/", 2)
             container_name = splitted[1]
             file_name = splitted[2]
-            if (
-                not container_name
-                or not file_name
-                or not os.path.basename(parsed_url.path)
-            ):
+
+            if parsed_url.scheme == "http":
```

Comment:
As I could stated it is similar to AWS where the URI starts with `aws://`. In this scenario, the `http` schema is only for covering the Azurite and works properly when a user tries to test it locally. Anyways, I'll add the test.

Reply:
Just added the tests.

Reply:
So the goal here is to ask people to use `http://` instead of `azure://` when it is Azurite, to be able to use a different logic to find the parts? What about adding an `azurite://` scheme instead?

Reply:
The schema `azurite://` is not identified locally, the Azurite Local Server is only accessed through `http://`, only if I map the `azurite://` schema and change it to `http://` on-the-fly.

Reply:
Yes, I think it is OK to make the replacement internally.

The main issue here, I think (and I might be wrong, so don’t blindly trust me on this and check, please), is that if a feed URL starts with `http://`, which has not been configured in [FEED_STORAGES](https://docs.scrapy.org/en/latest/topics/feed-exports.html#std-setting-FEED_STORAGES), it won’t work. And adding a generic `http` entry to `FEED_STORAGES` seems problematic to me. So I would instead document how `azurite` should be also added to `FEED_STORAGES` when using azurite.

If Azurite can be used with HTTPS instead of HTTP, you might need an extra option for users to specify that, e.g. `protocol = http|https` or `secure = True|False`.

Reply:
Done!